tools/data/hvu/preparing_hvu.md

```diff
@@ -43,15 +43,43 @@ More details can be found in [data_preparation](/docs/data_preparation.md)
 
 ## Step 4. Generate File List
 
-you can run the follow scripts to generate file list in the format of videos and rawframes, respectively.
+You can run the follow scripts to generate file list in the format of videos and rawframes, respectively.
 
 ```shell
 bash generate_videos_filelist.sh
 # execute the command below when rawframes are ready
 bash generate_rawframes_filelist.sh
 ```
 
-## Step 5. Folder Structure
+## Step 5. Generate File List for each individual tag categories.
```

Comment:
capitalization

Reply:
All 1st letters are capitalized except the conjunction `for`